test/controllers/api/v2/content_view_filter_rules_controller_test.rb

```diff
@@ -76,6 +77,12 @@ def test_show
       assert_template 'api/v2/content_view_filter_rules/show'
     end
 
+    def test_does_not_show
```

Comment:
The name of this test is a little ambiguous

How about something along the lines of `test_mismatched_filter_and_rule` which describes the test a bit better?